governance/votes/Executive Vote - November 12, 2021.md

```diff
@@ -0,0 +1,84 @@
+- [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam) parameter for WSTETH-A will be increased to 50 million DAI
+- Changes to the DIRECT-AAVEV2-DAI [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam), [Target Available Debt (gap)](https://makerdao.world/en/learn/governance/module-dciam), and Target Borrow Rate, as detailed below
+- 218,059.10 DAI will be returned to the Surplus Buffer from PauseProxy
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+$OfficeHours? This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### GUNIV3DAIUSDC-A Parameter Adjustments
+
+As per [this poll](https://vote.makerdao.com/polling/QmemHGSM?network=mainnet), the following GUNIV3DAIUSDC-A parameter adjustments will occur, if this executive passes:
+
+* Increase the [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam) from 50 million DAI to 500 million DAI
+* Decrease the [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio) from 105% to 102%
+* Decrease the [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee) from 1% to 0.5%
+
+For more information on this proposal, please see this [discussion thread](https://forum.makerdao.com/t/request-to-raise-the-guniv3daiusdc1-a-dc-to-500m/11394).
+
+### WSTETH-A Debt Ceiling Increase
+
+As per [this poll](https://vote.makerdao.com/polling/QmeQUKFm?network=mainnet), the following change will occur, if this executive passes:
+
+* Increase the [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam) from 5 million DAI to 50 million DAI
+
+For more information on this proposal, please see this [discussion thread](https://forum.makerdao.com/t/request-to-raise-staked-eth-dc-to-50m/11402).
+
+### DIRECT-AAVEV2-DAI Parameter Adjustments
```

Comment:
I'd be tempted to add (Aave D3M) or something here maybe? Not sure.

Reply:
Changed it - see what you think, can always change it back